sklearn/metrics/_regression.py

```diff
@@ -336,6 +341,10 @@ def mean_absolute_percentage_error(
     >>> y_pred = [2.5, 0.0, 2, 8]
     >>> mean_absolute_percentage_error(y_true, y_pred)
     0.3273...
+    >>> y_true = [1, 0, 2.4, 7]
+    >>> y_pred = [1.2, 0.1, 2.4, 8]
+    >>> mean_absolute_percentage_error(y_true, y_pred)
+    112589990684262.48
```

Comment:
May we move this to the end of this example and include an inline comment briefly describing what is happening here?